caddyhttp/push/setup_test.go

```diff
@@ -0,0 +1,272 @@
+package push
+
+import (
+	"net/http"
+	"reflect"
+	"testing"
+
+	"github.com/davecgh/go-spew/spew"
```

Comment:
I'm sure this is useful, but we try to keep our dependencies (even in tests) as lean as possible. Any chance this package could be replaced with an old faithful `fmt.Printf("%+v")` (or `%#v`) even though it's not quite as pretty?